tests/main/lxd-no-fuse/task.yaml

```diff
@@ -25,8 +25,15 @@ execute: |
 
     # The snapd package we build as part of the tests will only run on the
     # distro we build on. So we need to launch the right ubuntu version.
-    . /etc/os-release
-    lxd.lxc launch --quiet "ubuntu:${VERSION_ID}" my-ubuntu
+    (
+        . /etc/os-release
+        lxd.lxc launch --quiet "ubuntu:${VERSION_ID:-}" my-ubuntu
```

Comment:
Won't this fail on 20.04 too?

Reply:
Dunno, we'll see after https://github.com/snapcore/snapd/pull/8068 lands